web/src/lib/utils/storage-helper.test.tsx

```diff
@@ -0,0 +1,135 @@
+import { swrPrefixToIgnore, userDataPrefix, UserDataStorage, UseStorage } from "@/lib/utils/storage-helpers";
+import { generateUserData } from "@/test/factories";
+
+describe("storage-helper", () => {
```

Comment:
this file and its test should be camelCase instead of kebab case to fit our pattern